codegen/compiler/src/Quidditch/Target/Passes.td

```diff
@@ -29,6 +30,21 @@ def ConfigureForSnitchPass
   : InterfacePass<"quidditch-configure-for-snitch",
       "mlir::FunctionOpInterface">;
 
+def ConfigureTiles : InterfacePass<"quidditch-configure-tiles", "mlir::FunctionOpInterface"> {
+  let summary = "Annotate linalg operations with tile sizes";
+  let description = [{
+    Within each iree dispatch, annotate the root linalg operation with a tiling scheme (tile sizes + loop interchange).
+    Caveat: only tiles linalg operations of type matmul_transpose_b (for now)
+    Set the importTiles option to the path to the json file containing the tiling scheme for each dispatch.
+  }];
+  let options = [
+    Option<"importTiles", "import-tiles", "std::string", /*default=*/"",
+           "Name of a JSON file specifying loop bounds and order for each root linalg operation.">,
+    Option<"tablePointer", "NeverPassAValueHere", "std::uintptr_t", /*default=*/"0",
+           "Avoids opening the input file multiple times. Never pass a value to this option via the command line.">,
```

Comment:
Since the tiling schemes are passed in by the user in a single json file, it made sense to me that _before_ constructing the ConfigureTiles pass, that json file should be _read once_ and its information populated in a lookup table. This way, each time the ConfigureTiles pass runs over a functionOp, it doesn't have to open and read the contents of the same file again.
**I couldn't figure out an easy way to provide another argument to the ConfigureTiles pass without also exposing it to the user. Is it okay to leave it as is, or should I work harder and find a way to remove this as a command line argument?**